### ISSUE TITLE
feat: add unit tests to project

### ISSUE THREAD

**Issue by MrOsta** [Mr-Egil2/game-achievement#3]
`Unit testing`: This is a type of testing which is done by software developers in which the smallest testable module of an application - like functions, procedures or interfaces - are tested to ascertain if they are fit to use,

simply: some functions test other functions to confirm if it works at the required level without any errors

For example:
`index.js`
```javascript
// this function returns The sum of two numbers (parameters)
function sumTwoNumbers( number1, number2 ){
return ( number1 + number2 )
}

// call the function
sumTwoNumber( 2,3 );
// expected value: 5

```
- Ok how to confirm the expected value?
-  Basically: by using unit testing
   - ok but how to use unit testing?
      - using unit testing frameworks like `jest`

Ok, how do you use `jest` in the previous example?
It is very easy,

`test.js` 😄
```javascript


test("#sumTwoNumber function should return\output the sum of two numbers", function() {
  const output = sumTwoNumber( 2,3 )
  
  // expect() will do the test
  //  toBe() will check the value you expect ( you put it 2 and 3 so you expect it to be 5 )
  expect( output.toBe(5) )
});
```

```javascript

Thanks.
```
